delta-api/src/main/java/io/cdap/delta/api/DMLEvent.java

```diff
@@ -31,17 +31,21 @@
   private final StructuredRecord previousRow;
   private final String transactionId;
   private final long ingestTimestampMillis;
+  private final Long sourceTimestampMillis;
```

Comment:
I guess the reason you are not using a primitive long here is because you want to make it nullable , right ?
so how does a default value 0 hurt here for a primitive ?

Reply:
wanted to avoid additional storage cost with default value if any.. besides now I am using null check to assign the sequence number. please take a look again thanks!

Reply:
LGTM, thank you , Sagar :)